suite-native/formatters/src/TransactionIdFormatter.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import { Text, TextProps } from '@suite-native/atoms';
+
+type TransactionIdFormatterProps = {
+    transactionId: string;
+} & TextProps;
+
+export const TransactionIdFormatter = ({ transactionId, ...rest }: TransactionIdFormatterProps) => (
+    <Text variant="hint" color="gray1000" numberOfLines={1} ellipsizeMode="tail" {...rest}>
```

Comment:
Maybe put `{...rest}` before the numberOfLines and ellipsizeMode? You have it like that in`AccountAddressFormatter`. If you decide to do so, please also `Omit` the params numberOfLines and ellipsizeMode from the `TransactionIdFormatterProps`.

Reply:
I think this actually may be OK, because there could be place where do you want to be able to override these values.

Reply:
Ok then please just make it the same in `AccountAddressFormatter`. @juriczech

Reply:
https://github.com/trezor/trezor-suite/pull/7565/commits/f90ce6c4c6cc0efda17a2ec84ed89a449d76ace3